<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CsvEditor</class>
 <widget class="QMainWindow" name="CsvEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CsvEditor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="table"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>528</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuOptions">
     <property name="title">
      <string>Options</string>
     </property>
     <property name="icon">
      <iconset resource="qtgui.qrc">
       <normaloff>:/icons/setup.png</normaloff>:/icons/setup.png</iconset>
     </property>
     <addaction name="actionFirst_line_as_Headers"/>
    </widget>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="separator"/>
    <addaction name="menuOptions"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="separator"/>
  </widget>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="qtgui.qrc">
     <normaloff>:/icons/load.png</normaloff>:/icons/load.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="qtgui.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save...</string>
   </property>
  </action>
  <action name="actionFirst_line_as_Headers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>First line as Headers</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="qtgui.qrc"/>
 </resources>
 <connections/>
</ui>
